### PR TITLE
Explicitly declare HighDpiMode in the project file for integration test

### PR DESCRIPTION
**Pull request by Tanya-Solyanik**
... for easy experiments with DPI awareness.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7793)

### CODE REVIEW

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/WinformsControlsTest.csproj [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <EnableXlfLocalization>false</EnableXlfLocalization>
     <UpdateXlfOnBuild>false</UpdateXlfOnBuild>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ApplicationHighDpiMode>SystemAware</ApplicationHighDpiMode>
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/WinformsControlsTest.csproj:13]: We declare the settings globally here:
https://github.com/dotnet/winforms/blob/1b17d5b15bebc394af0a697c3ee0dffe285ae3b5/Directory.Build.props#L47-L50

So we should either declare the settings there, or move here.

Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/WinformsControlsTest.csproj:13]: I added Dpi mode to the test project in order to make it easier to experiment with this particular application. Global settings can stay unchanged.

Comment (RussKie) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/WinformsControlsTest.csproj:13]: ApplicationDefaultFont is there for the exact same purpose. So it's worth having all related options in the same place. I have no strong preference which place though.

Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/WinformsControlsTest.csproj:19]: I see, the root of confusion is that I have multiple test exes in my branch

**Review by RussKie — APPROVED** [review]

**Pull request merged**